homestead/php.ini

```diff
@@ -0,0 +1,5 @@
+xdebug.remote_autostart = 1
+xdebug.remote_host = 192.168.77.1
+xdebug.remote_connect_back = 0
+xdebug.coverage_enable=1
+memory_limit = -1
```

Comment:
Is this really needed?

Reply:
Thanks for your review @pablothedude. Let's grab a coffee real soon and discuss your feedback.

Reply:
I'm almost certain this is to prevent 'in-box' composer memory issues. @epinxteren is that assumption correct? 

If this is the case. I'd prefer to use the built-in composer memory limit features as described here: https://getcomposer.org/doc/articles/troubleshooting.md#memory-limit-errors